server.js

```diff
@@ -6038,6 +6039,8 @@ function versionColor(version) {
   var first = version[0];
   if (first === 'v') {
     first = version[1];
+  } else if (/^version\/[0-9]/.test(version)) {
+    version = 'v' + version.substring(8);
```

Comment:
Also, I believe you want `first = version[7]`. That said, I am fairly surprised by that change; does it happen in the wild? Could you give an example (on the separate PR)?

If we are at that point, it might be simpler to perform a single regex operation that fetches the first digit it can find in `version`.